modules/33acrossBidAdapter.js

```diff
@@ -143,14 +143,30 @@ function _createServerRequest(bidRequest, gdprConsent) {
 }
 
 // Sync object will always be of type iframe for TTX
-function _createSync(siteId) {
+function _createSync({siteId, gdprConsent = {}}) {
   const ttxSettings = config.getConfig('ttxSettings');
   const syncUrl = (ttxSettings && ttxSettings.syncUrl) || SYNC_ENDPOINT;
 
-  return {
+  const {consentString, gdprApplies} = gdprConsent;
+
+  const sync = {
     type: 'iframe',
     url: `${syncUrl}&id=${siteId}`
+  };
+
+  const gdprMap = {
+    'true': 1,
+    'false': 0
+  };
+
+  if (typeof gdprApplies === 'boolean') {
+    const gdpr = gdprMap[gdprApplies.toString()];
```

Comment:
you can do `Number(gdprApplies)`, as long as `gdprApplies` is boolean, this will either convert it to 0 or 1, so you do not need the `gdprMap`